lib/spack/spack/test/cmd/install.py

```diff
@@ -259,10 +259,41 @@ def test_install_from_file(spec, concretize, error_code, tmpdir):
     if concretize:
         spec.concretize()
 
-    with fs.working_dir(str(tmpdir)):
+    specfile = tmpdir.join('spec.yaml')
+
+    with specfile.open('w') as f:
+        spec.to_yaml(f)
+
+    # Relative path to specfile (regression for #6906)
```

Comment:
Quite nice idea. We should probably take the habit of noting down the issue if we implement a test for a regression.

Reply:
Began doing this after a couple of nightmare debugging nights both ended up in finding out that I was just introducing regressions. Low cost, high return!

Reply:
Cool! I was already thinking of marking slow tests to have something like:
```console
$ spack test -m "not slowtest"
```
running automatically in the background when I am developing. Marking also regressions makes a lot of sense imo.